crates/crates-io/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "crates-io"
-version = "0.36.1"
+version = "0.37.0"
```

Comment:
Side note: the version bump check in CI didn't work, since the new version in beta hasn't yet published. But in reality, the script should treat any version in beta as “published”.